tools/docs/github-query-items.ts

```diff
@@ -0,0 +1,16 @@
+export type GithubApiQueryResponse = {
```

Comment:
If we want to use "collapsible admonition blocks" like `??? note` then we should _first_ merge:

- https://github.com/renovatebot/renovatebot.github.io/pull/177